Code/GraphMol/Descriptors/Lipinski.h

```diff
@@ -29,15 +29,32 @@ const std::string lipinskiHBDVersion = "2.0.0";
 // bonds)
 unsigned int calcLipinskiHBD(const ROMol &mol);
 
+namespace NumRotatableBondsOptions {
```

Comment:
Any particular reason this isn't an enum?
Given that you had to do a bool overload anyway, it seems like it would make sense.
It would make the wrappers more rational too.